src/cider/nrepl/middleware/refresh.clj

```diff
@@ -1,10 +1,10 @@
-(ns cider.nrepl.middleware.refresh
+(ns ^{:clojure.tools.namespace.repl/load false
```

Comment:
Yeah, I was previously just using this locally when developing the middleware, since otherwise the refresh tracker state gets blown away every time I reload.

But since tools.namespace doesn't reload source in JARs, it shouldn't affect user code that depends on cider-nrepl (except for the case when cider-nrepl is a checkout dependency, but in that case you do want reloading of this namespace to be disabled - otherwise you only get full reloads, not incremental reloads, because the refresh tracker state is being blown away again).